docs/source/preparing_datasets/parallel_dataset_conversion.ipynb

```diff
@@ -435,7 +435,7 @@
                 "id": "NByc1ZGINcXe"
             },
             "source": [
-                "Let's checkout the root directories where you can see one `index.json` file and many shard files."
+                "Let's checkout the root directory, where you can see one `index.json` file and many shard files."
```

Comment:
overall is good. can you help change this sentence to "one `index.json` along with the subdirectories". (because shard files should not be present in the root folder.)

Reply:
done